bedrock/settings/base.py

```diff
@@ -1304,3 +1304,7 @@ def lazy_email_backend():
     CSP_IMG_SRC += (
         'ad.doubleclick.net',
     )
+
+# Bug 1340087 - Funnelcake experiments default to Windows 32bit and en-US builds only.
+FUNNELCAKE_PLATFORMS = config('FUNNELCAKE_PLATFORMS', default='win', cast=Csv())
```

Comment:
Looks to me like you do have this right, but you're welcome to add such a test or alter the existing test to use more than one value.